weaviate/batch/crud_batch.py

```diff
@@ -376,7 +377,7 @@ def add_data_object(
         self,
         data_object: dict,
         class_name: str,
-        uuid: Optional[str] = None,
+        uuid: Union[str, UUID, None] = None,
```

Comment:
If it is optional, use Optional[] instead of having None in the Union?

Reply:
`Optional` takes a single argument, not a list.
```python
TypeError: Optional[t] requires a single type.
```

Reply:
uhhh ok